src/Sylius/Behat/Context/Api/Admin/ManagingAdministratorsContext.php

```diff
@@ -91,6 +91,18 @@ public function iSpecifyItsNameAs(string $username = null): void
         }
     }
 
+    /**
+     * @When I specify its :field as too long string
+     */
+    public function iSpecifyItsFirstOrLastNameOrUsernameAs(string $field): void
+    {
+       match($field) {
+           'first name' => $this->client->addRequestData('firstName', $this->getTooLongString()),
+           'last name' => $this->client->addRequestData('lastName', $this->getTooLongString()),
+           'username' => $this->client->addRequestData('username', $this->getTooLongString()),
+       };
```

Comment:
```suggestion
       $this->client->addRequestData(lcfirst(trim(ucwords($field))), $this->getTooLingString());
```
though at that point you could remove the private function and just do the `str_repeat` in here.